python/hail/experimental/ldscore.py

```diff
@@ -71,107 +85,89 @@ def ld_score(entry_expr, annotation_exprs,
     entry_expr : :class:`.NumericExpression`
         Expression for entries of genotype matrix
         (e.g. ``mt.GT.n_alt_alleles()``).
+    locus_expr : :class:`.LocusExpression`
+        Row-indexed locus expression on a table or matrix table that is
```

Comment:
I think you should also mention in Notes section that if you have very large number of annotations, then rather than annotating the matrix table, it will be more efficient to import the annotations as a matrix table, ensure the rows are aligned with those of the genetic matrix table, and then write the block matrix `a` used in `l2 = r2_adj_sparse @ a` below.

Reply:
I added your first suggestion to a notes section. I think I'm going to address your second comment at a later date. That will require a bigger change to this function, and I'd like to see at what the performance is like with the current approach before redoing anything.